packages/ses/NEWS.md

```diff
@@ -1,14 +1,20 @@
 User-visible changes in `ses`:
 
-# Next release
+# v1.12.0 (2025-03-11)
 
-The `evalTaming:` option values are renamed
-- from `'safeEval'`, `'unsafeEval'`, and `'noEval'`
-- to `'safe-eval'`, `'unsafe-eval'`, and `'no-eval'`
+- The `evalTaming:` option values are renamed:
 
-in order to follow the convention that lockdown option values use kebob-case
-rather than camelCase. To avoid breaking old programs during the transition,
-the old names are deprecated, but continue to work for now.
+  - from `'safeEval'`, `'unsafeEval'`, and `'noEval'`
+  - to `'safe-eval'`, `'unsafe-eval'`, and `'no-eval'`
+
+  in order to follow the convention that lockdown option values use kebob-case
+  rather than camelCase. To avoid breaking old programs during the transition,
+  the old names are deprecated, but continue to work for now.
+
+- The value of expressions like `typeof unlikelyGlobal` is now `undefined`
+  instead of producing a `ReferenceError` because it proves impossible to
+  do so without revealing what properties exist on the host `globalThis`
+  to compartmentalized code.
```

Comment:
`typeof unlikelyGlobal` was already undefined before.

it's more about the case `try { unlikelyGlobal; } catch(e) { console.error(e); }` that would no longer print an error.